net.granjow.slowmovideo.yaml

```diff
@@ -0,0 +1,84 @@
+app-id: net.granjow.slowmovideo
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+x-add-extensions:
+    org.freedesktop.Platform.ffmpeg-full:
+        directory: lib/ffmpeg
+        version: "21.08"
+        add-ld-path: "."
+
+cleanup-commands:
+    - mkdir -p /app/lib/ffmpeg
+
+command: slowmoUI
+rename-icon: AppIcon
+rename-desktop-file: slowmoUI.desktop
+#rename-appdata-file: hugin.appdata.xml
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  # Because the file chooser somehow doesn't show the files on the host, we export some directory for reading pcap files
+  - --filesystem=xdg-public-share
+
+
+modules:
+
+  - name: opencv
+    # Taken from https://github.com/flathub/org.d3cod3.Mosaic/blob/be9aea900361940120341b2ad6fe7d97bec39c66/org.d3cod3.Mosaic.yml#L152
+    buildsystem: cmake-ninja
+    builddir: true
+    cleanup:
+        - '/include/'
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Debug
+      - -DOPENCV_GENERATE_PKGCONFIG=ON
+      #- -OPENCV_ENABLE_NONFREE=ON
+      - -DWITH_OPENCL=ON
+      - -DWITH_OPENGL=ON
+      - -DWITH_TBB=ON
+      - -DWITH_VULKAN=ON
+      - -DBUILD_WITH_DEBUG_INFO=OFF
+      - -DBUILD_TESTS=OFF
+      - -DBUILD_PERF_TESTS=OFF
+      - -DBUILD_EXAMPLES=OFF
+      - -DINSTALL_C_EXAMPLES=OFF
+      - -DINSTALL_PYTHON_EXAMPLES=OFF
+      # - -DOPENCV_EXTRA_MODULES_PATH="contrib/modules"
+      - -DWITH_1394=OFF
+      - -DWITH_PROTOBUF=ON
+      # - -DOPENCV_EXTRA_MODULES_PATH=../opencv_contrib-master/modules ../opencv-master
+      #- -DBUILD_LIST=calib3d,core,dnn,features2d,flann,highgui,imgcodecs,imgproc,java_bindings_generator,ml,objdetect,photo,python_bindings_generator,python_tests,stitching,video,videoio
+    sources:
+        - type: archive
+          url: https://github.com/opencv/opencv/archive/3.4.18.tar.gz
+          sha256: d6a69aa4a775a074662685b9bc2b07b88fb867f19d939b03ad281571b62f2620
+
+  - ffmpeg.yaml
```

Comment:
I see no justification for a separate file.

Reply:
cut and paste?

Reply:
sure. but copying a file is easier than locating the definition in the file and maintaining indentation.